pkg/access/access_token.go

```diff
@@ -0,0 +1,35 @@
+package access
+
+import (
+	"time"
+
+	"github.com/common-fate/ddb"
+	"github.com/common-fate/granted-approvals/pkg/storage/keys"
+	"github.com/common-fate/granted-approvals/pkg/types"
+)
+
+type AccessToken struct {
+	RequestId string `json:"request_id"` // maybe?
```

Comment:
Feedback on this section:
- everything we write to the DB needs to have a `dynamodbav` tag on all struct fields
- for JSON and dynamodbav tags we always use camelCase
- for Go variables we should never use `somethingId`, it should always be `somethingID` (the only case where this is acceptable in the codebase is in the auto-generated OpenAPI structs)

```suggestion
	RequestID string `json:"requestId" dynamodbav:"requestId"`
```